### PR TITLE
Bug/#31 Handle error when env file are missing

### PR DESCRIPTION
**Pull request by MangriMen**
Return value of tryConst is any, should we get rid of that?

### CODE REVIEW

**Review by FFreestyler — APPROVED** [review]

**Comment by MangriMen** [comment]
[Handle error when env file are missing](https://trello.com/c/y56rp26m/31-handle-error-when-env-file-are-missing)

File: src/helpers/common.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export function tryConst<T>(
```
Comment (IHiFiveI) [inline src/helpers/common.ts]: Maybe change naming to "tryCall"? Sounds more suitable for me

File: src/helpers/common.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  tryFunction: () => T,
+): ReturnType<typeof tryFunction> | any {
+  try {
+    return tryFunction();
+  } catch (e: any) {
```
Comment (IHiFiveI) [inline src/helpers/common.ts]: Doesn't error object have it's own type? smh like `string | {key: string, ... }`

File: src/helpers/firebase.ts [hunk 3]
```diff
@@ -13,4 +14,4 @@ const firebaseConfig = {
 
 const app = initializeApp(firebaseConfig);
 
-export const auth = getAuth(app);
+export const auth = tryConst(() => getAuth(app));
```
Comment (IHiFiveI) [inline src/helpers/firebase.ts]: I personally dislike the way it called (cant prove my point tho), but maybe change it to `tryConst(getAuth, app);`?

**Review by IHiFiveI — APPROVED** [review]

**Pull request merged**